bindings/java/java/src/test/java/org/ethereum/evmc/TestHostContext.java

```diff
@@ -0,0 +1,71 @@
+package org.ethereum.evmc;
+
+import java.nio.ByteBuffer;
+
+class TestHostContext implements HostContext {
+
+  @Override
+  public boolean accountExists(byte[] address) {
+    return true;
+  }
+
+  @Override
+  public ByteBuffer getStorage(byte[] address, byte[] key) {
+    return ByteBuffer.allocateDirect(64).put(new byte[64]);
+  }
+
+  @Override
+  public int setStorage(byte[] address, byte[] key, byte[] value) {
+    return 0;
+  }
+
+  @Override
+  public ByteBuffer getBalance(byte[] address) {
+    return ByteBuffer.allocateDirect(64).put(new byte[64]);
+  }
+
+  @Override
+  public int getCodeSize(byte[] address) {
+    return address.length;
+  }
+
+  @Override
+  public ByteBuffer getCodeHash(byte[] address) {
+    return ByteBuffer.allocateDirect(64).put(new byte[64]);
+  }
+
+  @Override
+  public ByteBuffer getCode(byte[] address) {
+    return ByteBuffer.allocateDirect(64).put(new byte[64]);
+  }
+
+  @Override
+  public void selfdestruct(byte[] address, byte[] beneficiary) {}
+
+  @Override
+  public ByteBuffer call(ByteBuffer msg) {
+
+    /*TestMessage message = new TestMessage(msg);
```

Comment:
Can you drop these comments (and similar comments in other places) if they are not needed?